modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,123 @@
+| Multiple persistent volumes per workspace
 
-[id="the-common-pvc-strategy_{context}"]
-== The `common` PVC strategy
+| Default
+| yes
+| no
+| no
 
-All workspaces inside a {platforms-namespace} use the same Persistent Volume Claim (PVC) as the default data storage when storing data such as the following in their declared volumes:
 
-* projects
-* workspace logs
-* additional Volumes defined by a use
+| Limitations
+| Maximum one running workspace per user when the persistent volume is in `ReadWriteOnce` access mode
+| Persistent volume count depends on the number of workspaces
+| Unpredictable persistent volume count
 
-When the `common` PVC strategy is in use, user-defined PVCs are ignored, and volumes related to these user-defined PVCs are replaced with a volume that refers to the common PVC.
-In this strategy, all {prod-short} workspaces use the same PVC. When the user runs one workspace, it only binds to one node in the cluster at a time.
+|===
 
-The corresponding containers volume mounts link to a common volume, and sub-paths are prefixed with `_<workspace-ID>_` or `__<original-PVC-name>__`. For more details, see xref:how-subpaths-are-used-in-pvcs_{context}[].
+Persistent volumes (PVs) access mode::
+The nature of the persistent volume determines the available access mode: `ReadWriteMany` or `ReadWriteOnce`.
+See link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - access mode].
+For example, link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] supports only the `ReadWriteOnce` access mode.
 
-The {prod-short} Volume name is identical to the name of the user-defined PVC. It means that if a machine is configured to use a {prod-short} volume with the same name as the user-defined
-PVC has, they will use the same shared folder in the common PVC.
 
-When a workspace is deleted, a corresponding subdirectory (`$\{ws-id}`) is deleted in the PV directory.
+[id="the-common-storage-strategy_{context}"]
+== The `common` storage strategy
 
-NOTE: The common PVC will be removed when a user's workspaces are all deleted. The PVC will be re-created when a non-ephemeral workspace is started.
+This is the default storage strategy.
+For each user, all workspaces use the same Persistent Volume (PV) for the default data storage.
 
-.Restrictions on using the `common` PVC strategy
+When the user starts a first non-ephemeral workspace, the workspace engine creates the common persistent volume.
 
-When the `common` strategy is used, and a workspace PVC access mode is ReadWriteOnce (RWO), only one node can simultaneously use the PVC.
+When the user starts a non-ephemeral workspace, the workspace engine uses the common persistent volume.
+The workspace only binds simultaneously to one node in the {orch-name} cluster.
+The workspace engine ignores user-defined volumes. 
+The workspace engine replaces volumes related to user-defined volumes with a subpath in the common volume. 
```

Comment:
The 2 rows above are contradictory. First we say that user-defined volumes are ignored. Then that they are "replaced" with a subpath in the common volume.

The "common" PVC strategy is implemented using [subPaths](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) of a "common to all workspaces" PV. That's not specific of user defined volumes but applies to any file that needs to be persisted through the restarts of a workspace (projects included).